event/serializers.py

```diff
@@ -1,31 +1,61 @@
 from rest_framework import serializers
 from images.serializers import ImageSerializer
-from .models import Event, OcialClient, Like
+from user.serializers import UserSerializer
+from .models import Event, Like
+from ocial.models import Category
 
 
-class OcialClientSerializer(serializers.ModelSerializer):
-
+class LikeSerializer(serializers.ModelSerializer):
     class Meta:
-        model = OcialClient
-        fields = ["name", "defaultLatitude", "defaultLongitude"]
-
+        model = Like
+        fields = "__all__"
+class EventSerializer(serializers.Serializer):
+    creator = UserSerializer()
+    name = serializers.CharField()
+    place = serializers.CharField()
+    description = serializers.CharField()
+    timeStart = serializers.DateTimeField()
+    timeEnd = serializers.DateTimeField()
+    capacity = serializers.IntegerField()
+    category = serializers.ChoiceField(
+        choices=[(category.value, category.name) for category in Category],
+        default=Category.SPORTS.value, 
+        required=False)
+    latitude = serializers.FloatField()
+    longitude = serializers.FloatField()
+    image = ImageSerializer(read_only=True)
+    created_at = serializers.DateTimeField()
+    updated_at = serializers.DateTimeField()
+    likes = serializers.SerializerMethodField()
+
+    def get_likes(self, instance):
+        return instance.get_likes_count()
+
+    def to_representation(self, instance):
+        event_data = {
+            "creator": UserSerializer(instance.ocialClient, context={'user': self.context['user']}).data,
+            "name": instance.name,
+            "place": instance.place,
+            "description": instance.description,
+            "timeStart": instance.timeStart,
+            "timeEnd": instance.timeEnd,
+            "capacity": instance.capacity,
+            "category": instance.category,
+            "latitude": instance.latitude,
+            "longitude": instance.longitude,
+            "image": ImageSerializer(instance.image).data,
+            "created_at": instance.created_at,
+            "updated_at": instance.updated_at,
+            "likes": self.get_likes(instance),
```

Comment:
```suggestion
    def to_representation(self, instance):
        event_data = {
            "creator": UserSerializer(instance.ocialClient, context={'user': self.context['user']}).data,
            "name": instance.name,
            "place": instance.place,
            "description": instance.description,
            "timeStart": instance.timeStart,
            "timeEnd": instance.timeEnd,
            "capacity": instance.capacity,
            "category": instance.category,
            "latitude": instance.latitude,
            "longitude": instance.longitude,
            "image": ImageSerializer(instance.image).data,
            "created_at": instance.created_at,
            "updated_at": instance.updated_at,
            "likes": instance.get_likes_count(),
```

Esto lo tienes que dejar así para que el OpenAPI se genere sin warnings

Reply:
Hecho.